README.md

```diff
@@ -72,7 +73,6 @@ releasify publish [--path|-p <path>]             ➡ The path to the project to
                   [--gh-group-by-label|-l <label>] ➡ Group the commits in the changelog message by PR's labels
                   [--npm-access|-a <string>]       ➡ It will set the --access flag of `npm publish` command. Useful for scoped modules. The value must be [public, restricted]
                   [--npm-dist-tag <string>]        ➡ It will add a npm tag to the module, like `beta` or `next`
-                  [--npm-otp <code>]               ➡ It will provide the otp code to the npm publish
```

Comment:
Could you leave this argument?

If wee keep it, we could still think to use releasify automation within optic https://github.com/nearform/optic#usage

Reply:
will do